backfill/shared/keys.go

```diff
@@ -0,0 +1,32 @@
+package shared
+
+import (
+	"fmt"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/makerdao/vdb-mcd-transformers/backfill/repository"
+	"github.com/makerdao/vdb-mcd-transformers/transformers/storage/utilities"
+	"github.com/makerdao/vulcanizedb/libraries/shared/storage"
+)
+
+func GetIlkArtKey(ilk string) common.Hash {
+	return storage.GetKeyForMapping(storage.IndexTwo, ilk)
+}
+
+func GetUrnArtKey(u repository.Urn) (common.Hash, error) {
```

Comment:
I wonder if it's worth making [getUrnArtKey](https://github.com/makerdao/vdb-mcd-transformers/blob/staging/transformers/storage/vat/keys_loader.go#L268) and [getUrnInkKey](https://github.com/makerdao/vdb-mcd-transformers/blob/61e6bf6da0b9b25715632fae10afddda9b59f489/transformers/storage/vat/keys_loader.go#L259) public so that we can reuse that code. If the way we get either of these keys changes, it would be nice to only have to change it in one place. Though I'm not sure how likely it is that that will ever change. 🤔